[2025-02-15T01:40:46.304+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.transform_daily manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:40:46.318+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.transform_daily manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T01:40:46.320+0000] {taskinstance.py:1331} INFO - Starting attempt 2 of 2
[2025-02-15T01:40:46.342+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): transform_daily> on 2025-02-15 00:56:13.638531+00:00
[2025-02-15T01:40:46.349+0000] {standard_task_runner.py:57} INFO - Started process 163 to run task
[2025-02-15T01:40:46.353+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'transform_daily', 'manual__2025-02-15T00:56:13.638531+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpbm004yv6']
[2025-02-15T01:40:46.357+0000] {standard_task_runner.py:85} INFO - Job 80: Subtask transform_daily
[2025-02-15T01:40:46.478+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.transform_daily manual__2025-02-15T00:56:13.638531+00:00 [running]> on host 184396a03990
[2025-02-15T01:40:46.671+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='transform_daily' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:56:13.638531+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:56:13.638531+00:00'
[2025-02-15T01:40:46.828+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T01:40:46.855+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=transform_daily, execution_date=20250215T005613, start_date=20250215T014046, end_date=20250215T014046
[2025-02-15T01:40:46.891+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T01:40:46.929+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
